tests/js/spec/views/performance/landing/utils.spec.jsx

```diff
@@ -38,15 +38,15 @@ describe('Utils', function () {
       const projects = [TestStubs.Project()];
       const data = initializeData(projects, {landingDisplay: 'frontend_pageload'});
       expect(getCurrentLandingDisplay(data.router.location, projects).label).toEqual(
-        'Frontend (Pageload)'
+        'Web Vitals'
```

Comment:
this util functions were used in the new landing page but they were operating with old `LANDING_DISPLAYS`. With new `LANDING_DISPLAYS`, I've updated assertions on labels